mxnet-bot/LabelBotFullFunctionality/deploy_bot.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 # Script to deploy the label bot to our dev account
 set -e
-
 export AWS_PROFILE=mxnet-ci-dev
+
+dest_account="106780517423"
+bot_account="968277166688"
+
+sed "s/###HERE###/$dest_account/g" log-access-trust-policy.json-template > log-access-trust-policy.json
```

Comment:
Nice hack!

Reply:
Actually I will update this to leave those account spots blank

Reply:
I would use python formatting, but ok.